### DISCUSSION TITLE
Custom domain

### DISCUSSION THREAD

**Discussion by EvilZoidYT** [plan-player-analytics/Plan#2548] [Q&A] [answered]
Hi there! I want to use a custom domain for the website. Is there anyway?

**Comment by AuroraLS3** [comment, marked as answer]
Set up an A-record in your DNS

Then change Alternative_IP settings in Plan to match

**Reply by EvilZoidYT** [reply]
I kind of not understand. Oh btw things i forgot to tell: i use cloudflare and i want to use a subdomain.

**Reply by AuroraLS3** [reply]
### A-record example:

plan.example.com  1.2.3.4 (The IP of the server) TTL: Auto

Then set Plan `Webserver.Alternative_IP` settings to `plan.example.com:%port%` (Don't replace `%port%`) And access Plan from http://plan.example.com:8804 (If using default port)

**Reply by EvilZoidYT** [reply]
Yep works now thanks
